src/components/Dropdown/Dropdown.spec.tsx

```diff
@@ -167,7 +167,7 @@ describe('Components / Dropdown', () => {
   });
 
   describe('Dropdown item render', async () => {
-    it('should override Dropdown.Item base component when using `as` prop', async () => {
+    it('should override Dropdownn.Item base component when using `as` prop', async () => {
```

Comment:
Keep the latest changes here

Reply:
fixed in https://github.com/themesberg/flowbite-react/pull/1298/commits/3292c56f764797bd21f38aa399e590ff2553de59